tests/test_resources/test_clusters/test_on_demand_cluster.py

```diff
@@ -201,6 +201,36 @@ def test_fn_to_docker_container(self, ondemand_aws_cluster):
     ####################################################################################################
     # Status tests
     ####################################################################################################
+
+    @pytest.mark.level("minimal")
+    @pytest.mark.clustertest
+    def test_status_scheduler_basic_flow(self, cluster):
+
+        cluster_uri = rh.globals.rns_client.format_rns_address(cluster.rns_address)
+        api_server_url = rh.globals.rns_client.api_server_url
+        headers = rh.globals.rns_client.request_headers()
+
+        # TODO [sb]: call cluster's /logs endpoint once introduced.
+        cluster_logs = cluster.run([f"cat {SERVER_LOGFILE_PATH}"], stream_logs=False)[
+            0
+        ][1]
+
+        assert (
+            f'HTTP Request: POST {api_server_url}/resource/{cluster_uri}/cluster/status "HTTP/1.1 200 OK"'
```

Comment:
similar note as below

Reply:
updated this test accordingly as well. (see comment below)